src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const MALE = 'm';
```

Comment:
male what? only by implementation will another developer understand that this is male sex
so better rename to `MALE_SEX`